test/catalyst/assetAttributesRegistry/fixtures.ts

```diff
@@ -29,19 +35,19 @@ export async function setCatalyst(
 }> {
   const {
     assetAttributesRegistry,
-    assetAttributesRegistryAdmin,
+    assetUpgrader,
   } = await setupAssetAttributesRegistry();
   if (collectionId) {
     await waitFor(
-      assetAttributesRegistry
-        .connect(ethers.provider.getSigner(assetAttributesRegistryAdmin))
-        .setCatalyst(collectionId, catalystId, gemsIds)
+      assetUpgrader
```

Comment:
@wighawag That's where we started using assetUpgrader to test assetAttributesRegistry

Reply:
Thanks @CloverGuy 
I think it is fine to use asset upgrader as setCatalyst is not meant to be called by any EOA.
So if we consider the whole with assetUpgrader and asserUpdated as input, these new test are fine.

But I guess your point is that we could have kept the older test and add new one.
I agree that it would have been a good idea.
Now that the PR is merged, I think the best would be to re-add the old test in a new PR.
If that is quite a lot of work, I would focus on adding test that are not covered yet and then look at that later